cpp/Platform.Timestamps/UniqueTimestampFactory.h

```diff
@@ -1,14 +1,13 @@
 ﻿namespace Platform::Timestamps
 {
-    class UniqueTimestampFactory : public IFactory<Timestamp>
+    class UniqueTimestampFactory
     {
-        private: std::uint64_t _lastTicks = 0;
-
-        public: Timestamp Create()
+    private: std::uint64_t _lastTicks = 0;
```

Comment:
If this file created translator then add tab(4 spaces) before public/private/protected